src/DynamicData/List/ObservableListEx.cs

```diff
@@ -1713,7 +1713,7 @@ public static IObservable<IChangeSet<T>> FlattenBufferResult<T>(this IObservable
         /// <exception cref="System.ArgumentNullException">source</exception>
         public static IObservable<IChangeSet<T>> BufferIf<T>([NotNull] this IObservable<IChangeSet<T>> source,
```

Comment:
There doesn't seem to be a version with just the `pauseIfTrueSelector` and maybe the scheduler?

Reply:
It's just hiding unmodified on L1696